.changelogs/issue_2237.yml

```diff
@@ -0,0 +1,6 @@
+significance: patch
+type: changed
+comment: Added a helper function for returning precision with filter for
+  floating values in the reporting section for quizzes and assignment's average
+  column.
+entry: Added helper function for precision for floating values in reporting section.
```

Comment:
I would see the name of the function here.
The entry test will appear in the plugin's CHANGELOG.md and in the release note's post, e.g.:
https://make.lifterlms.com/2023/03/23/lifterlms-version-7-1-2/

You can also add a reference to the new filter added.

Reply:
Also you can link this entry to the issue 2237.

Additionally, I would add another changelog entry (so another file) because we're changing the default precision on the quizzes table.

While you're there can you check where else we're using that 3 digits rounding?

Reply:
Done. I've created the separate changelog in the lifterlms-assignments repo, where it belongs.